kytos/utils/client.py

```diff
@@ -78,6 +78,43 @@ def get_napp(self, username, name):
                                  username, name, res.status_code, res.reason)
         return json.loads(res.content)
 
+    def reload_napp(self, username, napp):
+        """Reload a NApp.
+
+        Raises:
+            requests.HTTPError: When there's a server error.
+
+        """
+        api = self._config.get('napps', 'api')
+        endpoint = os.path.join(api, 'napps', username, napp, '')
+        response = self.make_request(endpoint)
+
+        if response.status_code == 404:  # We need to know if NApp is not found
+            return None
+        elif response.status_code != 200:
+            raise KytosException('Error getting %s/%s from server: (%d) - %s',
+                                 username, napp, response.status_code,
+                                 response.reason)
+        return json.loads(response.content)
+
+    def reload_all_napps(self):
+        """Reload all NApps.
+
+        Raises:
+            requests.HTTPError: When there's a server error.
+
+        """
+        api = self._config.get('napps', 'api')
+        endpoint = os.path.join(api, 'napps', 'reload', 'all')
+        response = self.make_request(endpoint)
+
+        if response.status_code == 404:  # We need to know if NApp is not found
+            return None
+        elif response.status_code != 200:
+            raise KytosException('Error getting %s/%s from server: (%d) - %s',
+                                 response.status_code, response.reason)
+        return json.loads(response.content)
```

Comment:
reload_napp() is almost equal to reload_all_naps(). They differ only by one line, which is a great indicator that they should be merged into one method.

Reply:
Okay, I'll do it!